java/gtnh/egg-greg-tech--new-horizons.json

```
@@ -17,7 +17,7 @@
         "Java 21": "ghcr.io\/pterodactyl\/yolks:java_21"
     },
     "file_denylist": [],
-    "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -Dfml.readTimeout=180 @java9args.txt -jar lwjgl3ify-forgePatches.jar nogui",
+    "startup": "java -Xms{{SERVER_MEMORY}}M -Xmx{{SERVER_MEMORY}}M -Dfml.readTimeout=180 @java9args.txt -jar lwjgl3ify-forgePatches.jar nogui",
```

This crashes for me after world has been created. Did some testing and this might be our best shot at pinning the heap size without doing some calculations with the server size:
```suggestion
    "startup": "java -XX:InitialRAMPercentage=95.0 -XX:MaxRAMPercentage=95.0 -Dfml.readTimeout=180 @java9args.txt -jar lwjgl3ify-forgePatches.jar nogui",
```